libshvbroker/src/aclmanager.cpp

```diff
@@ -305,14 +305,22 @@ void AclManager::setGroupForLdapUser(const std::string_view& user_name, const st
 	m_ldapUserGroups.emplace(user_name, group_name);
 }
 #endif
-
-chainpack::RpcValue AclManager::accessGrantForShvPath(const std::string& user_name, const shv::core::utils::ShvUrl &shv_url, const std::string &method, bool is_request_from_master_broker, bool is_service_provider_mount_point_relative_call, const shv::chainpack::RpcValue &rq_grant)
-{
-	logAclResolveM() << "==== accessGrantForShvPath user:" << user_name << "requested path:" << shv_url.toString() << "method:" << method << "request grant:" << rq_grant.toCpon();
+chainpack::AccessGrant AclManager::accessGrantForShvPath(
+		const std::string &user_name,
+		const core::utils::ShvUrl &shv_url,
+		const std::string &method,
+		bool is_request_from_master_broker,
+		bool is_service_provider_mount_point_relative_call,
+		const chainpack::AccessGrant &access_grant
+)
+{
+	using AccessLevel = chainpack::MetaMethod::AccessLevel;
```

Comment:
```suggestion
	using chainpack::MetaMethod::AccessLevel;
```

Reply:
![image](https://github.com/silicon-heaven/libshv/assets/4949019/a203b03d-d157-484c-ac10-125494657100)

I've got this error

Reply:
Yeah, it doesn't work like that, my mistake.

We should create a header for AccessLevel, and put it there. Users of the headers can use `using chainpack::MetaMethod::AccessLevel;` on their own.